shared/utils/profile-stats.py

```diff
@@ -342,6 +355,15 @@ def show_profile_stats(self, profile, options):
                     self.console_print(profile_stats['missing_anaconda_fixes'],
                                        console_width)
 
+                if profile_stats['missing_stig_ids']:
+                    print("*** rules of '%s' profile missing "
+                          "STIG IDs: %d of %d have them [%d%% missing]"
+                          % (profile, rules_count - missing_stig_ids_count,
+                             rules_count,
+                             (100.0 * missing_stig_ids_count / rules_count)))
+                    self.console_print(profile_stats['missing_stig_ids'],
+                                       console_width)
+
```

Comment:
Missing STIG IDs are only printed when `missing_fixes` option is set, I'd expect there would be `missing_stig_ids` option.